test/ex_tldr_test.exs

```diff
@@ -58,4 +63,12 @@ defmodule ExTldrTest do
 
     assert capture_io(execute_main) =~ "Usage:"
   end
+
+  #test "should return lack of internet" do
+    # assert_raise NoInternetConnectionError,
+    #             fn ->
+    #               ClientMock
+    #               |> expect(:get, fn _ -> {:error, %HTTPoison.Error{reason: :nxdomain}} end)
+    #             end
+  #end
```

Comment:
This is the test I referred in my last comment. I think that this one applied to a mock of a behavior defined for our wrapper, and not for `HTTPoison.Base`, will work (if am not wrong...).